cmd/exec/operations.go

```diff
@@ -450,40 +450,59 @@ func executeEvent(url string) (string, error) {
 	}
 
 	defer inputDataOsFile.Close()
-	byteResult, _ := ioutil.ReadAll(inputDataOsFile)
-	var event map[string]interface{}
-	json.Unmarshal([]byte(byteResult), &event)
+	byteResult, err := ioutil.ReadAll(inputDataOsFile)
+
+	if err != nil {
+		return "", err
+	}
 
-	// Read input data as bytes.buffer
-	inputData, err := safeLoadFile(localAbsPath)
+	var event map[string]interface{}
+	err = json.Unmarshal([]byte(byteResult), &event)
```

Comment:
I'm okay with this as-is, however, it might be worth using the official cloudevents library to load and parse the file. That could get some helpful error messages happening at this step.

Reply:
extracting keys from bytes.buffer type was very difficult, so I think i will read it as bytes.buffer to test if file is correct but not using it